tests/scripts/components-build-system.sh

```diff
@@ -207,3 +207,21 @@ component_build_cmake_programs_no_testing () {
 support_build_cmake_programs_no_testing () {
     support_test_cmake_out_of_source
 }
+
+# For timebeing, no VIA Padlock platform available.
+component_build_aes_via_padlock () {
```

Comment:
This is like aesce (for the time being): it's a platform-specific feature that we are only building, not testing, because we don't have suitable hardware to run it on the CI.